Sandbox/ZergRushBot.cs

```diff
@@ -14,17 +14,14 @@ namespace Sandbox
     public class ZergRushBot : IBot
     {
         // Every time there are this many idle soldiers, attack
-        private const uint AttackThreshold = 24;
-
-        private const uint MaxWorkersPerMineralDeposit = 2;
-
-        private Dictionary<ulong, List<ulong>> workersByMineralDeposit;
+        private const int AttackThreshold = 16;
```

Comment:
This number seems to be the difference between "getting punked" and "getting pretty solidly beaten" by Benchmark Bot. 

Zerglings. They don't make 'em like they used to.